src/services/SessionEventProcessor.ts

```diff
@@ -92,5 +115,26 @@ export class SessionEventProcessor {
 			throw new AppError(HttpCodesEnum.SERVER_ERROR, error.message);
 		}
 	}
+
+	async sendOldTemplateEmail(oldSessionEvent: SessionEvent): Promise<void> {
```

Comment:
Is this stuff for backwards compatibility and so will be eventually deprecated, or is not having post office information a valid scenario we can expect in the future?